app/controllers/admin/conference_controller.rb

```diff
@@ -67,11 +66,16 @@ def create
 
       if @conference.valid?
```

Comment:
If you touch this action anyway, what about refactor this to 
``
@conference.save
``
instead of
``
if @conference.valid?
  @conference.save
  ...
``

it's done twice ...